README.md

```diff
@@ -88,6 +88,18 @@ Run `carthage update --cache-builds` then drag & drop from the Carthage/Builds f
 - `RxNimbleRxBlocking.framework` and `RxBlocking.framework`
 - `RxNimbleRxTest.framework` and `RxTest.framework`
 
+## Migration 4.5.0 -> 5.0.0
+
+Deprecated function `equalFirst` were completely removed in favor of more `Nimble`-like style of API. Please use:
+
+```swift
+expect(o).first == ...
```

Comment:
should we also add `expect(o).first.to(equal(...))` syntax? I know some teams prefer `==`, but afaik `to(equal(...))` is also valid syntax and more consistent because `== nil` can't be used (`to(beNil())` must be used instead)

Reply:
Yeah, we can add that, even though that knowledge is more specific to Nimble but I don't mind. Matchers usage is not specific to RxNimble.

Reply:
Right, but that’s another reason to use to(equal(...)) as that’s the first syntax suggested by Nimble’s readme. I mean, if we’ll suggest only one, why would we suggest the == one?

Reply:
Fixed, thanks